### PR TITLE
Node guide changes

### PR DESCRIPTION
**Pull request by vaibhavchellani**

### CODE REVIEW

File: deploy-migrations/README.md [hunk 1]
```diff
@@ -1,31 +1,56 @@
...
 npm run truffle:compile
 ```
 
-4. Check account that you are deploying from has ropsten ether.
+### :four: Deploy contracts
+
+We need to deploy our set of contracts on 2 chains:
+
+- Base Chain: Ideally a higher security EVM chain which can be used for dispute resolution. For testing ganache or any other EVM chain should work.
+- Child Chain: EVM compatible chain to work as our sidechain. For testing ganache or any other EVM chain should work.
```
[the remark this answers is not in the document]
Comment (vaibhavchellani) [reply deploy-migrations/README.md]: Got it, will update. Ganache for base-chain is fine, right?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "testrpc": "ganache-cli -k constantinople --mnemonic 'clock radar mass judge dismiss just intact mind resemble fringe diary casino'",
+    "testrpc": "ganache-cli -k constantinople --mnemonic 'clock radar mass judge dismiss just intact mind resemble fringe diary casino' --gasLimit 13000000 -g 0",
```
[the remark this answers is not in the document]
Comment (vaibhavchellani) [reply package.json]: Sure

File: truffle-config.js [hunk 3]
```diff
@@ -1,27 +1,27 @@
-require('babel-register')
-require('babel-polyfill')
+require("babel-register");
```
[the remark this answers is not in the document]
Comment (vaibhavchellani) [reply truffle-config.js]: @0xAshish can you update this file please.

File: deploy-migrations/README.md [hunk 4]
```diff
@@ -1,31 +1,56 @@
...
+| bor_testnet2 |     https://testnet2.matic.network      |       8995 |
+| development  |          http://localhost:8545          |         \* |
+| matic_dev    |          http://localhost:8546          |         \* |
+
+Feel free to add your own. Update the chain url in `networks.matic` key in [truffle-config.js](../truffle-config.js).
+
+### :two: Export variables
+
 ```
+
```
[the remark this answers is not in the document]
Comment (0xAshish) [reply deploy-migrations/README.md]: @atvanguard this should be fine he has just added EOF.

**Comment by vaibhavchellani** [comment]
Resolved all comments.

File: deploy-migrations/README.md [hunk 5]
```diff
@@ -1,31 +1,55 @@
 # Deploy Contracts
-1. Use Infura to point to the main chain node (Ethereum ropsten)
+
+### :one: We have some network configrations available.
+
+| Network Name |                   URL                   | Network ID |
+| ------------ | :-------------------------------------: | ---------: |
+| ropsten      | https://ropsten.infura.io/v3/${API_KEY} |          3 |
+| mainnet      | https://mainnet.infura.io/v3/${API_KEY} |          1 |
+| bor_testnet2 |     https://testnet2.matic.network      |       8995 |
```
[the remark this answers is not in the document]
Comment (vaibhavchellani) [reply deploy-migrations/README.md]: Updated by @0xAshish

**Pull request merged**